public/js/plugins/github/github-modal.controller.js

```diff
@@ -48,14 +52,17 @@ module.exports =
       }
     });
 
-    // Fill the paginatedRepos array with the first page of repos.
-    vm.onPageChange();
-
     return vm.repos;
   }
 
   function fetchRepos(name) {
-    githubService.fetchRepos(name).then(setRepos);
+    if (name) {
+      vm.org_name = name
+    }
```

Comment:
Set the org name globally here so we can call `fetchRepos` in contexts where we don't have the org name like in like 110.